apps/wpcom-block-editor/src/calypso/features/iframe-bridge-server.js

```diff
@@ -311,15 +313,15 @@ function handleUpdateImageBlocks( calypsoPort ) {
 		attrNames = { ...attrNames, id: 'id', url: 'url' };
 
 		if ( 'deleted' === image.status ) {
-			dispatch( 'core/editor' ).updateBlockAttributes( block.clientId, {
+			dispatch( blockEditorStore ).updateBlockAttributes( block.clientId, {
 				[ attrNames.id ]: undefined,
 				[ attrNames.url ]: undefined,
 			} );
 			return;
 		}
 
 		preloadImage( image.url ).then( () => {
-			dispatch( 'core/editor' ).updateBlockAttributes( block.clientId, {
+			dispatch( blockEditorStore ).updateBlockAttributes( block.clientId, {
```

Comment:
Did you mean `editorStore`?

```suggestion
			dispatch( editorStore ).updateBlockAttributes( block.clientId, {
```